mongodbatlas/containers.go

```diff
@@ -107,7 +140,8 @@ func (s *ContainersServiceOp) Get(ctx context.Context, groupID, containerID stri
 	return root, resp, err
 }
 
-// Add a network peering container to the project associated to {GROUP-ID}.
+// Create adds a network peering container to the project associated to {GROUP-ID}.
+//
```

Comment:
A lot of these are to fix rendering here https://pkg.go.dev/go.mongodb.org/atlas/mongodbatlas?tab=doc#ContainersServiceOp.Create